src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -476,6 +492,9 @@ private Task CloseAsync(ConnectionContext connection)
             // REVIEW: If a token is passed in and is canceled during FlushAsync it seems to break .Complete()...
             await _connectionState.Connection.Transport.Output.FlushAsync();
 
+            // we've sent a message, so don't ping for a while
```

Comment:
nit: In SignalR we start comments with capitals. Don't ask me why, it was the standard when I started 😄

Reply:
I'm not even sure we're that consistent :)